Tasks/PunishmentTasks.cs

```diff
@@ -57,6 +57,38 @@ public static async Task<bool> CheckMutesAsync()
                 }
 #if DEBUG
                 Program.discord.Logger.LogDebug(Program.CliptokEventID, "Checked mutes at {time} with result: {result}", DateTime.Now, success);
+#endif
+                return success;
+            }
+        }
+
+        public static async Task<bool> CheckAutomaticWarningsAsync()
+        {
+            if (Program.cfgjson.AutoWarnMsgAutoDeleteDays == 0)
+                return false;
+            Dictionary<string, UserWarning> warnList = Program.db.HashGetAll("automaticWarnings").ToDictionary(
+                x => x.Name.ToString(),
+                x => JsonConvert.DeserializeObject<UserWarning>(x.Value)
+            );
+            if (warnList is null | warnList.Keys.Count == 0)
```

Comment:
Newlines to improve readability of this bit, primarily so its clear where the `if` ends.
```suggestion
            if (Program.cfgjson.AutoWarnMsgAutoDeleteDays == 0)
                return false;

            Dictionary<string, UserWarning> warnList = Program.db.HashGetAll("automaticWarnings").ToDictionary(
                x => x.Name.ToString(),
                x => JsonConvert.DeserializeObject<UserWarning>(x.Value)
            );

            if (warnList is null | warnList.Keys.Count == 0)
```